src/test/java/org/apache/commons/lang3/RegExUtilsTest.java

```diff
@@ -379,4 +381,149 @@ public void testReplacePatternDeprecated() {
         assertEquals("Lorem_ipsum_dolor_sit", RegExUtils.replacePattern("Lorem ipsum  dolor   sit", "( +)([a-z]+)", "_$2"));
     }
 
+    private RegExUtils utils;
+
+    @BeforeEach
+    void setUp() {
+        utils = new RegExUtils();
+    }
+
+    @Test
+    public void testFindMultipleMatches() {
```

Comment:
Can you use parameterized logging instead of multiple methods with the same logic?